cluster/manifests/z-karpenter/karpenter.k8s.aws_awsnodetemplates.yaml

```diff
@@ -194,6 +194,59 @@ spec:
                       credentials are not available."
                     type: string
                 type: object
+              networkInterfaces:
+                items:
+                  properties:
+                    associateCarrierIPAddress:
+                      description: "Associates a Carrier IP address with eth0 for
+                        a new network interface. \n Use this option when you launch
+                        an instance in a Wavelength Zone and want to associate a Carrier
+                        IP address with the network interface. For more information
+                        about Carrier IP addresses, see Carrier IP addresses (https://docs.aws.amazon.com/wavelength/latest/developerguide/how-wavelengths-work.html#provider-owned-ip)
+                        in the Wavelength Developer Guide."
+                      type: boolean
+                    associatePublicIPAddress:
```

Comment:
Don't we need to explicitly set this option somewhere in the configuration as well?

Reply:
yea true, added it cluster/node-pools/worker-karpenter/provisioners.yaml